ui/app/store/actions.test.js

```diff
@@ -1793,4 +1793,52 @@ describe('Actions', function () {
       assert(background.unMarkPasswordForgotten.calledOnce);
     });
   });
+
+  describe('#displayWarning', function () {
```

Comment:
Was this stuff accidentally added in a rebase or something? :thinking:

Reply:
nah, i moved them from files in the test unit that didn't match real files <_<

Reply:
this one came from 'test/unit/actions/warning.test.js'

Reply:
the tx one from 'tx.test.js' in same folder

Reply:
Oh I see now - good catch! They were hidden behind a "Load diff" that I hadn't noticed.